ja/lessons/basics/basics.md

```diff
@@ -211,6 +211,8 @@ iex> not 42
 ** (ArgumentError) argument error
 ```
 
+参考: Elixirの`and`と`or`はErlangの`andalso`と`orelse`に実際に対応しています。
```

Comment:
don't we need to translate `which doesn't evaluates first argument and evaluates the second argument.` ?

Reply:
removed? https://github.com/elixirschool/elixirschool/commit/22a033dc7fe90fe9a49e51983b7ab493b7246b43#diff-56389d258534fde2065909c908684202

Reply:
oops